patches/chromium/fix_check_issecureeventinputenabled_in_constructor_before_setting.patch

```diff
@@ -0,0 +1,21 @@
+From 0000000000000000000000000000000000000000 Mon Sep 17 00:00:00 2001
+From: John Kleinschmidt <jkleinsc@github.com>
+Date: Thu, 9 Jul 2020 11:16:30 -0400
+Subject: fix: check IsSecureEventInputEnabled in constructor before setting
+ SetPasswordInputEnabled to true
+
+It appears that macOS 12 has IsSecureEventInputEnabled set to true initially.
```

Comment:
this is upstreamed already, right?

Reply:
Yes - https://chromium-review.googlesource.com/c/chromium/src/+/2295749

Reply:
Yes but we haven't absorbed it yet